tests/test_modeling_common.py

```diff
@@ -2814,6 +2818,10 @@ def test_model_from_pretrained_torch_dtype(self):
         # test auto-detection
         model = AutoModel.from_pretrained(TINY_T5, torch_dtype="auto")
         self.assertEqual(model.dtype, torch.float32)
+        # XXX: some models don't have `torch_dtype` in `config.json` so we can't test those at the moment
```

Comment:
leaving this as a reminder that we need to add torch_dtype to at least the models we control.